modules/light-clients/07-tendermint/client_state.go

```diff
@@ -286,11 +286,13 @@ func (cs ClientState) VerifyNonMembership(
 		return sdkerrors.Wrap(clienttypes.ErrConsensusStateNotFound, "please ensure the proof was constructed against a height that exists on the client")
 	}
 
-	if err := merkleProof.VerifyNonMembership(cs.ProofSpecs, consensusState.GetRoot(), merklePath); err != nil {
-		return err
+	// if absenceSentinelValue is defined, the counterparty has a sentinel absence value stored for the nonexistent key
+	// thus we can prove membership of this sentinel value to prove absence of the key in state.
+	// if the value is not defined, then we verify a nonmembership proof
+	if cs.AbsenceSentinelValue != nil {
```

Comment:
Instead of accessing the value directly (i.e. `.AbsenceSentinelValue`), should't we add a getter method to the [ClientState interface](https://github.com/cosmos/ibc-go/blob/main/modules/core/exported/client.go#L36) that all implementations of the interface can then fulfill?